packages/data-context/src/sources/migration/codegen.ts

```diff
@@ -205,16 +207,8 @@ function formatObjectForConfig (obj: Record<string, unknown>) {
 }
 
 function createE2eTemplate (pluginPath: string, hasPluginsFile: boolean, options: Record<string, unknown>) {
-  const requirePlugins = `return require('./${pluginPath}')(on, config)`
```

Comment:
I'm not sure if the problem here is with synchronous requires, as much as that the file at the `pluginsPath` has `export default`, and our old child process logic automatically handled that case. 

Wondering if we just want to check if the file we're referencing here as `pluginPath` contains `export default`, and if it does we would  tell the migrator to write out `require('./${pluginPath}').default(on, config)` or something 🤔